doc/tooling/tcm/tcm_access_control/tcm_ldap_auth.rst

```diff
@@ -0,0 +1,150 @@
+of LDAP users in |tcm| are defined by LDAP group mapping.
+
+Both LDAP and secure LDAPS (LDAP over TLS) protocols are supported.
+
+..  _tcm_ldap_auth_enable:
+
+Enabling LDAP authentication
+----------------------------
+
+To allow LDAP user authentication in |tcm|, enable the ``ldap`` authentication method
+in the :ref:`security.auth <tcm_configuration_reference_security_auth>` configuration option before startup:
+
+-   In the YAML |tcm| configuration:
+
+    .. code-block:: yaml
+
+        security:
+          auth:
+            - ldap
+
+-   In the command line:
+
+    .. code-block:: console
+
+        $ tcm --security.auth="ldap"
+
+.. note::
+
+    If both authentication methods -- LDAP and local -- are enabled, |tcm| tries them
+    for each login attempt in the order they are specified in the configuration.
+
+..  _tcm_ldap_auth_config:
+
+LDAP configuration
+------------------
+
+To enable LDAP user access to |tcm|, create an *LDAP configuration* that connects
+|tcm| to the LDAP server that stores the users. An LDAP configuration
+defines how |tcm| connects to the server and queries user data. To create a LDAP
+configuration, go to the **LDAP** page and click **Add**.
+
+To edit a LDAP configuration, click **Edit** in the **Actions** menu of the corresponding row.
+
+To delete a LDAP configuration, click **Delete** in the **Actions** menu of the corresponding row.
+
+..  _tcm_ldap_auth_config_general:
+
+General settings
+~~~~~~~~~~~~~~~~
+
+Define the general configuration settings:
+
+*   **Enabled**. Defines if the configuration is used. Turn the toggle off to
+    stop using the configuration.
+
+    .. note::
+
+        If there are several enabled LDAP configurations, |tcm| attempts to use them
+        for user authentication in the order they are created.
+
+*   **Automatically add non-existent users**. By default, |tcm| automatically saves
+    LDAP user information to its :ref:`backend store <tcm_backend_store>`
+    upon their first login. Turn the toggle off if you don't want to save users from this LDAP server.
+
+..  _tcm_ldap_auth_config_connect:
+
+LDAP server connection
+~~~~~~~~~~~~~~~~~~~~~~
+
+Enter the LDAP server connection parameters:
+
+*   **Endpoints**. URLs of the LDAP server.
+*   **Request timeout**. The timeout for |tcm| requests to the LDAP server, in seconds.
+*   **Enabled TLS**. If the server uses LDAPS, turn this toggle on and specify
+    TLS connection parameters, such as a certificate and a key file.
+
+..  _tcm_ldap_auth_config_query:
+
+LDAP queries
+~~~~~~~~~~~~
+
+To define how |tcm| queries the LDAP server for user authentication and authorization,
+fill in the fields of the **Queries** step:
+
+-   **Query user** and **Query password**. Credentials of the LDAP user on behalf
+    of which all LDAP queries are executed: a distinguished name (DN) and a password.
+    Example DN: ``cn=admin,cn=users,dc=tarantool,dc=io``.
```

Comment:
Nit: IMO, such long examples can be shown in a separate code block for better readability:

```suggestion
-   **Query user** and **Query password**. Credentials of the LDAP user on behalf
    of which all LDAP queries are executed: a distinguished name (DN) and a password.
    Example DN:
    
    ..  code-block:: text
    
        cn=admin,cn=users,dc=tarantool,dc=io
```

The same for other samples. For instance, the `Group query template` example doesn't fit into one line on my laptop :)